cucumber/api/features/change_password.feature

```diff
@@ -13,6 +13,16 @@ Feature: Change the password of a role
     Given I set the password for "alice" to "My-Password1"
     When I successfully PUT "/authn/cucumber/password" with username "alice" and password "My-Password1" and plain text body "New-Password1"
     Then I can GET "/authn/cucumber/login" with username "alice" and password "New-Password1"
+    And there is an audit record matching:
+    """
+      <84>1 * * conjur * password
+      [auth@43868 user="cucumber:user:alice"]
```

Comment:
should this be changed to `role` as well? it can also have a host

Reply:
No. Under `auth` section the field name is expected to be user. Only under subject it's expected to be role

Reply:
do we want to change it to `role` for uniformity? it does make more sense. @netacoral wdyt?

Reply:
The goal of this PR is to have a single format, yes. The alignment we want to have is `user` under `auth` structured data and 'role' under `subject` structured data. (@egvili maybe you should add it to the PR description).
So @orenbm to your question, in this case it should be 'user'.

Reply:
i agree that it is out of scope of this PR but i think it is drunk to have `user` in one place and `role` in another where in both places we are referring to a role (user/host). I'm not saying we should do it now, just asking if you think that the correct behaviour is to have `role` in both places. If so, we should create an issue to fix it (unless there's a reason not to)

Reply:
Actually the way I saw it is that 'user' is the identity performing the operation and 'role' is the identity the operation is performed on, so it's ok they're not the same to avoid confusion. But I see your point, and I'm not sure what's the best approach...